releasenotes/notes/bug-fix-multiplexer-qubits-654bb6f8ece25b02.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixes a bug with the `"multiplexer"` simulator instruction where the
```

A single "`" is italics, I assume you wanted a code block like:

```suggestion
    Fixes a bug with the ``"multiplexer"`` simulator instruction where the
```